recipes/libevent/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2.0.22" %}
+
+package:
+  name: libevent
+  version: {{ version }}
+
+source:
+  fn: libevent-{{ version }}.tar.gz
+  url: https://github.com/libevent/libevent/archive/release-{{ version }}-stable.tar.gz
+  sha256: ab89639b0819befb1d8b293d52047c6955f8d1c9150c2b22a0e6247930eb9128
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - toolchain
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - openssl 1.0.*
+    - python
+
+  run:
+    - openssl 1.0.*
+    - python
```

Comment:
So this ties back into the [convo above]( https://github.com/conda-forge/staged-recipes/pull/1507#r77994202 ). Basically if we are installing the Python script, it only seems to be supported for Python 2.

Though personally I would be ok with removing that script and then dropping Python as dependency. I don't know that any one actually uses it. This way it should be more or less compatible with the conda version.

Side note: I broke `libevent` out into PR ( https://github.com/conda-forge/staged-recipes/pull/1491 ). This merely stacked on top of it.